rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtilsTest.java

```diff
@@ -822,6 +822,10 @@ else if (CovidConstants.FormFields.PATIENT_INFO_DOB.equals(field.getString(JsonF
                 numOfPopulatedFields++;
             }
         }
+        else if (CovidConstants.FormFields.SAMPLER_NAME.equals(field.getString(JsonFormConstants.KEY))) {
+            assertEquals(field.getString(VALUE), "");
```

Comment:
We should assert that this value is actually pre-populated with a non-empty String.

Reply:
fixed.